src/content/pt/2019/ecommerce.md

```diff
@@ -0,0 +1,639 @@
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vROHKGuMutXmxtzEoBSJNVn7DOzlfpizJh7mOkopFK8OVl_hCUHDOmKgYOojrpVsGnGWaletE7Uc5oX/pubchart?oid=323146848&format=interactive"
+  )
+}}
+
+{{ figure_markup(
+  image="fig13.png",
+  caption="Distribuição de solicitações de imagem por página de comércio eletrônico.",
+  description="Distribution of the 10, 25, 50, 75, and 90th percentiles of image requests per ecommerce page. The median desktop ecommerce page makes 40 image requests. The 10th percentile is 16 requests, 25th 25, 75th 62, and 90th 97. The desktop distribution is slightly higher than mobile by 2-10 requests at each percentile.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vROHKGuMutXmxtzEoBSJNVn7DOzlfpizJh7mOkopFK8OVl_hCUHDOmKgYOojrpVsGnGWaletE7Uc5oX/pubchart?oid=1483037371&format=interactive"
+  )
+}}
+
+<p class="note">Observe que, como nossa <a href="./methodology">metodologia</a> de coleta de dados não simula as interações do usuário em páginas como clicar ou rolar, as imagens com carregamento lento não seriam representadas nesses resultados.</p>
+
+As Figuras 13.12 e 13.13 acima mostram que a página média de comércio eletrônico tem 37 imagens e uma carga útil de imagens de 1.517 KB no celular, 40 imagens e 1.524 KB no desktop. 10% das páginas iniciais têm 90 ou mais imagens e uma carga útil de imagem de quase 6 MB!
+
+{{ figure_markup(
+  caption="O número médio de bytes de imagem por página de comércio eletrônico para celular.",
+  content="1,517 KB",
+  classes="big-number"
+)
+}}
+
+Uma proporção significativa de páginas de comércio eletrônico tem cargas úteis de imagem consideráveis ​​e faz um grande número de solicitações de imagem no primeiro carregamento. Consulte o relatório de <a hreflang="en" href="https://httparchive.org/reports/state-of-images">estado de imagens</a> do HTTP Archive's e os capítulos de [mídia](./media) e [peso de página](./page-weight) para obter mais contexto.
+
+Os proprietários de sites desejam que seus sites tenham uma boa aparência em dispositivos modernos. Como resultado, muitos sites oferecem as mesmas imagens de produtos de alta resolução para todos os usuários, _ndependentemente da resolução ou do tamanho da tela_. Os desenvolvedores podem não estar cientes (ou não querer usar) técnicas responsivas que permitem a entrega eficiente da melhor imagem possível para diferentes usuários. Vale lembrar que imagens de alta resolução podem não necessariamente aumentar as taxas de conversão. Por outro lado, o uso excessivo de imagens pesadas pode afetar a velocidade da página e, portanto, _reduzir_ as taxas de conversão. Na experiência dos autores em análises e eventos de sites, alguns desenvolvedores e outras partes interessadas têm SEO ou outras preocupações sobre o uso de carregamento lento para imagens.
```

Comment:
I think in the "_ndependentemente da resolução ou do tamanho da tela_" phrase is missing the `i` letter.

I believe the phrase "ou não querer usar" could be more fluid with "ou não querem usar"

Reply:
> I think in the "_ndependentemente da resolução ou do tamanho da tela_" phrase is missing the `i` letter.
> 
> I believe the phrase "ou não querer usar" could be more fluid with "ou não querem usar"

Thanks